### PR TITLE
fixed login and added a second protected page

### PR DESCRIPTION
**Pull request by afyacy**
- The sign-in works now.
- Added second protected page.
- Protected pages [/home & /dashboard]

### CODE REVIEW

File: pages/dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react'
+import { useSession } from 'next-auth/client'
+import router from 'next/router'
+
+export default function Dashboard () {
+  const [session] = useSession()
+
+  useEffect(() => {
```
Comment (simbamentor) [inline pages/dashboard/index.js:8]: nice solution 👍

File: pages/login.js [hunk 2]
```diff
@@ -1,93 +1,68 @@
...
-    if (result.error !== null) {
-      if (result.status === 401) {
-        resultError = 'Your username/password combination was incorrect. Please try again'
-      } else {
-        resultError = result.error
-      }
-    } else {
+  const loginUser = async (data) => {
+    try {
+      // eslint-disable-next-line no-unused-vars
```
Comment (simbamentor) [inline pages/login.js:11]: you don't need to disable this

it's also possible to make the request without assigning it to a variable, like this:

```
await signIn('credentials', {
        redirect: false,
        email: data.email,
        password: data.password
      })
```

**Review by simbamentor — CHANGES_REQUESTED** [review]
looks good, added some comments

File: pages/login.js (hunk 2 above)
Comment (afyacy) [reply pages/login.js]: done

**Pull request merged**